corehq/apps/export/templates/export/spec/ko/mocha.html

```diff
@@ -10,6 +10,5 @@
 {% block mocha_tests %}
 <script src="{% static 'export/spec/data/export_instances.js' %}"></script>
 <script src="{% static 'export/spec/ExportInstance.spec.js' %}"></script>
-<script src="{% static 'export/spec/ExportColumn.spec.js' %}"></script>
```

Comment:
Oh actually I just figured it out I think. I believe that `ExportColumn.spec.js` was actually being executed for me locally, because the file still exists in my staticfiles dir, and the test was failing. So the error wasn't that the file was missing, the error was that a test in that file was failing.